ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/Utils.java

```diff
@@ -196,6 +196,11 @@ public static boolean shouldReplicate(ReplicationSpec replicationSpec, Table tab
         }
         return shouldReplicateExternalTables;
       }
+
+      // Skip dumping events related to ACID tables if bootstrap is enabled on it
```

Comment:
shouldReplicate is not checked by events such as AllocateWriteIdEvent and CommitTxnEvent.
CommitTxnEvent cannot be skipped but need to remove all AcidWriteEvents which are packed along with it.

Reply:
Do you mean WriteEventInfo instead of AcidWriteEvents? I believe CommitTxnEvent can contain only write events for ACID tables. So writeEventInfoList in CommitTxnHandler.handle() should be set to null.

Reply:
yes, that's what I meant.

Reply:
Done.